packages/bsky/src/api/app/bsky/feed/getPostThread.ts

```diff
@@ -36,6 +36,7 @@ export default function (server: Server, ctx: AppContext) {
       const hydrateCtx = await ctx.hydrator.createContext({
         labelers,
         viewer,
+        includeThreadRootAuthorState: true,
```

Comment:
The main reason `includeTakedowns` and `include3pBlocks` are context options is that they are fairly far-reaching through the hydration and view call stacks.  The usage of `includeThreadRootAuthorState` is quite localized though.  If it needs to stick around, I wonder if it could become an option/argument of the hydration method where it's used.